modules/airgap/setup/private_registry.sh

```diff
@@ -0,0 +1,46 @@
+set -x
```

Comment:
you dont need to pass the shell here ?

also add a comment on waht the script is doing, accomplish

Reply:
Updated